### PR TITLE
Disk profiles

### PR DESCRIPTION
**Pull request by freshfrmthehood**
Added the ability to list all disk_profiles for a storage domain, and add a disk profile as on option when creating a volume.

### CODE REVIEW

**Comment by mmoll** [comment]
@freshfrmthehood what's the status here?

**Comment by abenari** [comment]
@freshfrmthehood thanks for your code contribution.
I have fixed last comment, rebased and merged your changes. You are welcome to test it!
Thanks,
  Amos.

**Comment by freshfrmthehood** [comment]
Hello,

You'll be surprised about what I've found on the web, just take a look <http://berespiho.evtradinpost.com/lnszzsr>

All the best, Doug Bishop

**Comment by lzap** [comment]
@freshfrmthehood whoops you should change your password immediately. Sorry, but I reported this as an abuse and then realized it's your account. I hope github support will just reset the password.

File: lib/ovirt/disk_profiles.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+module OVIRT
+  class DiskProfile < BaseObject
```
Comment (dougbishop) [inline lib/ovirt/disk_profiles.rb]: Hey friend, 


There are a couple of books that inspire me to do really great  things,  here is  the list of  that books <http://practice.goscrappin.com/lnjsdo>

Kind regards, Doug Bishop

**Comment by dougbishop** [comment]
Hey friend, 

I found  something  absolutely new and so interesting! Don't miss  it  up, you may find more  info here <http://high.virtualcourt.org/lnzmo>

Doug Bishop

File: lib/ovirt/disk_profiles.rb (hunk 1 above)
Comment (dougbishop) [inline lib/ovirt/disk_profiles.rb]: Hey,  I was looking for some  stuff  on the web and found this,  you're gonna  like it, just  take a look here <http://lonatheno.gkaindustries.com/lnpsnpj>

Warm regards, DOTSON SCOTT

**Pull request merged**